index.js

```diff
@@ -14,9 +14,18 @@
 //. [![CircleCI](https://img.shields.io/circleci/project/github/sanctuary-js/sanctuary/master.svg)](https://circleci.com/gh/sanctuary-js/sanctuary/tree/master)
 //. [![Gitter](https://img.shields.io/gitter/room/badges/shields.svg)](https://gitter.im/sanctuary-js/sanctuary)
 //.
-//. Sanctuary is a functional programming library inspired by Haskell
-//. and PureScript. It's stricter and more opinionated than [Ramda][],
-//. and provides a similar suite of functions.
+//. Sanctuary is a JavaScript functional programming library inspired by
+//. Haskell and PureScript. It's stricter than [Ramda][], and provides a
+//. similar suite of functions.
+//.
+//. Sanctuary promotes programs composed of simple, pure functions. Such
+//. programs are not just a pleasure to write; they are easy to comprehend,
```

Comment:
Great suggestion @Jacse. I think `only` reads better than `just` in this kind of construction @davidchambers. And we may yet flip it around: `Such functions are not only easier to comprehend, test, and maintain; they are also a pleasure to write.` I always like the "reward" to appear at the end of the sentence.